src/scss/includes/panels/service_panel.scss

```diff
@@ -133,22 +134,23 @@
   }
 }
 
-.service_panel__action__icon {
-  span {
-    font-size: 23px;
-    line-height: 36px;
-  }
-}
-
 @media (max-width: 640px){
 
   .panel.service_panel {
-    height: 215px;
+    height: 210px;
     background: #fff;
+
+    hr {
+      margin-bottom: 16px;
+    }
+
+    .panel-content {
+      height: calc(100% - 26px);
+    }
   }
 
   .panel.service_panel .panel-resizeHandle {
-    min-height: 50px;
+    min-height: 25px;
```

Comment:
I think that this change makes the handle significantly harder to grab, and causes missed touches.

Also (I am not sure it's related), in somes cases the panel content will scroll during the unfold, which is unexpected.
![Peek 08-04-2020 11-18](https://user-images.githubusercontent.com/4726554/78768370-20bf1f00-798c-11ea-901d-ca84fc7e51ad.gif)

Reply:
I don't reproduce the weird scroll effect in any browser, pretty sure it's unrelated to a couple of sizes modified in CSS though :)
OK: I'll make the grab area bigger.